### PR TITLE
[Antora] [PGSQL] Add Operate section to postgresql doc

### PR DESCRIPTION
**Pull request by vttranlina**

### CODE REVIEW

**Review by Arsnael — APPROVED** [review]
Overall LGTM :)

File: docs/modules/servers/partials/operate/logging.adoc [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+review of performances at the tier level.
+
+Leveraging filters in Kibana discover view can help to filter out
+''already known'' frequently occurring logs.
+
+When reporting ERROR or WARNING logs, consider adding the full logs, and
+related data (eg the raw content of a mail triggering an issue) to the
+bug report in order to ease resolution.
+
+== Logging configuration
```
[the remark this answers is not in the document]
Comment (vttranlina) [reply docs/modules/servers/partials/operate/logging.adoc:20]: Created a task for it https://github.com/linagora/james-project/issues/5220

**Comment by vttranlina** [comment]
squash & resolve conflict

File: docs/modules/servers/pages/postgres/operate/webadmin/admin-mail-queues-extend.adoc [hunk 2]
```diff
@@ -0,0 +1 @@
+// The document only covers Postgres
```
Comment (quantranhong1999) [inline docs/modules/servers/pages/postgres/operate/webadmin/admin-mail-queues-extend.adoc:1]: Only cover Cassandra/Distributed to be more correct?

**Review by quantranhong1999 — APPROVED** [review]

File: docs/modules/servers/pages/postgres/operate/webadmin/admin-mail-queues-extend.adoc (hunk 2 above)
Comment (vttranlina) [reply docs/modules/servers/pages/postgres/operate/webadmin/admin-mail-queues-extend.adoc:1]: My mean, this file is meant to cover Postgres documentation but is currently empty.

**Review by Arsnael — APPROVED** [review]

**Review by hungphan227 — APPROVED** [review]

**Pull request merged**